pyproject.toml

```diff
@@ -1,26 +1,36 @@
 [tool.black]
-line-length = 100
+line-length = 120
 target-version = ['py38']
-include = 'server,tests/server'
+
+[tool.darker]
+revision = 'origin/dev...'
+color = true
+isort = true
+
+[tool.isort]
+profile = "black"
+known_third_party = ["pytest"]
 
 [tool.pylint]
-    [tool.pylint.'MESSAGES CONTROL']
-    max-line-length = 100
-    disable = [
-        'logging-format-interpolation',
-        # Allow pytest functions to be part of a class
-        'no-self-use',
-        'too-many-locals',
-        'too-many-arguments',
-        # Allow pytest classes to have one test
-        'too-few-public-methods',
-    ]
+[tool.pylint.main]
+max-line-length = 120
+disable = [
+    'logging-format-interpolation',
+    # Allow pytest functions to be part of a class
+    'no-self-use',
+    'too-many-locals',
+    'too-many-arguments',
+    'too-many-branches',
+    # Allow pytest classes to have one test
+    'too-few-public-methods',
+]
+enable = 'useless-suppression'
 
-    [tool.pylint.'BASIC']
-    # Allow arbitrarily short-named variables.
-    variable-rgx = ['[a-z_][a-z0-9_]*']
-    argument-rgx = [ '[a-z_][a-z0-9_]*' ]
-    attr-rgx = ['[a-z_][a-z0-9_]*']
+[tool.pylint.basic]
+# Allow arbitrarily short-named variables.
+variable-rgx = '[A-Za-z_][a-z0-9_]*'
+argument-rgx = '[A-Za-z_][a-z0-9_]*'
+attr-rgx = '[A-Za-z_][a-z0-9_]*'
```

Comment:
i reserve the right to complain about these (or lack of them) as we use this more!

Reply:
Fair! Tbh, as per the comment, I think the main reason for this is we had some mathy code that used mathy variables like `X` or `W` for matrix operations and the linter complained. So we wound up with this essentially YOLO arbitrary names regex.

A possible alternative solution is to upgrade to [Pylint 3.0.0](https://pylint.readthedocs.io/en/latest/whatsnew/3/3.0/index.html#changes-requiring-user-actions):
> The invalid-name message no longer checks for a minimum length of 3 characters by default. (This was an unadvertised commingling of concerns between casing and name length, and users regularly reported this to be surprising.)

I could try bumping pylint in covidcast-indicators and if 3.0.0 passes there, we could probably then pin in both repos.

Reply:
So upgrading pylint in `covidcast-indicators` produces a lot of new errors that will take a while to fix. I think we should just keep both `delphi-epidata` and `covidcast-indicators` at `pylint==2.8.3` for now, since that will allow us to bring this repo up incrementally and that will already be a nice improvement. After a bit of that, we can think about upgrading pylint and addressing the new issues. We can also tighten up these naming conventions in a PR that addresses just naming.